release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -760,3 +767,9 @@ URIs use the `git://` or `ssh://` scheme.
 the mirror registry only links to the builder service account. The pull secret
 must also link to the build config object.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1810904[*BZ#1810904*])
+
+* When a user selects a node or a group of nodes in the *Topology* view on the Firefox browser, the backgrounds of all the associated labels and nodes become transparent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822337[*BZ#1822337*])
+
+* In  the *Topology view*, when a user selects a node or workload and then clicks the *View monitoring dashboard* link in the *Monitoring* tab on the side panel, the user sees the monitoring dashboard for that specific workload. This filtered workload dashboard view is not clearly named, causing confusion with the generic dashboard that displays metrics for all the workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822331[*BZ#1822331*])
+
+* In the *Developer* perspective, when invalid characters such as a period (.) are entered in the serverless traffic distribution tag, it stops working, but displays no error messages to prevent invalid characters from being entered in the tag. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822344[*BZ#1822344*])
```

Comment:
"it stops working," - please clarify what "it" is.  I can't tell if you mean the developer perspective of the distribution tag.

No comma after working.

This sentence is too long.

Reply:
Fixed the changes and have shortened the sentence, let me know if this is alright. Thank you!